tests/kafkatest/services/streams.py

```diff
@@ -477,6 +477,10 @@ def __init__(self, test_context, kafka):
                                                                  "")
         self.UPGRADE_FROM = None
         self.UPGRADE_TO = None
+        self.extra_properties = {}
+
+    def set_config(self, key, value):
+        self.extra_properties[key] = value
```

Comment:
I've added this as a general mechanism in a couple of places to pass specific configs into Streams, so we don't have to make new constructors for every different parameterization.